system-int-test/test/cmr/system_int_test/search/collection_search_format_test.clj

```diff
@@ -695,6 +687,42 @@
                                            {:url-extension "json"})
                 [:status :results])))))))
 
+(deftest search-collection-csv
+  (let [c1 (d/ingest "PROV1" (dc/collection {:short-name "shortie number 1"
+                                              :version-id "V1"
+                                              :long-name "lng nme"
+                                              :processing-level-id "L1"
+                                              :entry-title "I am #1"
+                                              :beginning-date-time "1970-01-01T12:00:00Z"
+                                              :platforms [{:short-name "platform #1"
+                                                           :long-name "platform #1"
+                                                           :type "t"}]}))
+        c2 (d/ingest "PROV2" (dc/collection {:short-name "world's shortest name: so short you won't believe your eyes!"
+                                              :version-id "V2"
+                                              :long-name "world's longest name"
+                                              :processing-level-id "L2"
+                                              :entry-title "lorem ipsum"
+                                              :beginning-date-time "2000-01-01T12:00:00Z"
+                                              :platforms [{:short-name "platform #21"
+                                                           :long-name "platform #21"
+                                                           :type "good platform"}
+                                                          {:short-name "platform #32"
+                                                           :long-name "platform #32"
+                                                           :type "very good platform"}]}))
+        g1 (d/ingest "PROV1" (dg/granule c1))])
+  (index/wait-until-indexed)
+
+  (let [response (search/find-concepts-csv :collection {:include-granule-counts "true"})]
+    (is (= 200 (:status response)))
+    (is (= (str "Data Provider,Short Name,Version,Entry Title,Processing Level,Platforms,Start Time,End Time\n"
+                "PROV1,,V1,I am #1,L1,platform #1,1970-01-01T12:00:00.000Z,\nPROV2,,V2,lorem ipsum,L2,\"platform #21,platform #32\",2000-01-01T12:00:00.000Z,\n")
```

Comment:
Looks like the `Short Name` is not exported correctly in the csv result.

Reply:
Fixed